python/test_model/basic_types/__init__.py

```diff
@@ -0,0 +1,39 @@
+# This file was generated by the "yardl" tool. DO NOT EDIT.
+
+# pyright: reportUnusedImport=false
+from typing import Tuple as _Tuple
+import re as _re
+import numpy as _np
+
+_MIN_NUMPY_VERSION = (1, 22, 0)
```

Comment:
Small thing: we could skip some of the stuff in this file if it's an imported package

Reply:
Absolutely, fixed.